homeworks/02-image-layout/src/App.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styles from './App.module.css'
+
+const MIN_HEIGHT = 200
+
+interface Props {}
+interface State {
+  loadedCount: number
+}
+
+class App extends React.Component<Props, State> {
+
+  state = {
+    loadedCount: 0
+  }
+  images: HTMLImageElement[] = []
+
+  componentDidMount() {
+    for (let i = 1; i <= 35; i++) {
+      const image = new Image()
+      image.onload = this.imageLoaded.bind(this)
+      image.src = require(`./images/${i}.jpg`).default
+      this.images.push(image)
+    }
+  }
+
+  imageLoaded() {
+    this.setState(state => ({loadedCount: state.loadedCount + 1}))
```

Comment:
I don't see, where do you use this counter in the render function?

Reply:
It's not used, it's just a way to force re-render.